.mvn/maven.config

```diff
@@ -0,0 +1 @@
+-Pconsume-incrementals
```

Comment:
you should not need this for this PR?
if you do then the version you are depending on has not been released and then this should make this project unreleasable

Reply:
and I don't see the extension to use that.

Reply:
Correct, fixed now